src/pybind/mgr/dashboard/frontend/cypress/integration/orchestrator/workflow/02-create-cluster-add-host.feature

```diff
@@ -0,0 +1,41 @@
+@login
+Feature: Cluster expansion host addition
+
+    Add some hosts and perform some host related actions like editing the labels
+    and removing the hosts from the cluster and verify all of the actions are performed
+    as expected
+
+    Background: Cluster expansion wizard
+        Given the welcome screen
+        Then go to 'wizard'
+        Then go to 'Add Host' section
+
+    Scenario Outline: Confirm there is an existing host, then add and remove hosts ceph-node-01 and ceph-node-02
+        Then verify host 'ceph-node-00' 'exists'
+        Then add host '<hostname>' with labels '<labels>'
+        Then remove host '<hostname>'
+
+        Examples:
+            | hostname | labels |
+            | ceph-node-01 ||
+            | ceph-node-02 ||
+
+    Scenario Outline: Add hosts using pattern 'ceph-node-[01-02]'
+        Then add host '<hostname>' using pattern
```

Comment:
What's the user-visible outcome of this? This is rather a `When` step rather than a `then` (`then` has to be the description of an effect/outcome, not a task).
```suggestion
        When the user adds a host pattern '<hostname>'
```